.github/actions/set-env-variables/action.yml

```diff
@@ -12,7 +12,7 @@ runs:
         echo "EGRESS_GATEWAY_HELM_VALUES=--helm-set=egressGateway.enabled=true" >> $GITHUB_ENV
         echo "CILIUM_CLI_RELEASE_REPO=cilium/cilium-cli" >> $GITHUB_ENV
         # renovate: datasource=github-releases depName=cilium/cilium-cli
-        CILIUM_CLI_VERSION="v0.16.19"
+        CILIUM_CLI_VERSION="v0.16.21"
```

Comment:
Shouldn't that be `v.0.16.22` @aanm ? Or am I missing something

Reply:
Should be fixed by the latest force push. The release was not published on github.com/cilium/cilium-cli yet, that's why still the previous release v0.16.21 was picked up.

Reply:
Ok thanks for the heads up